pipeline/schemas/token.py

```diff
@@ -40,6 +40,10 @@ class TokenPatch(Patchable):
     name: Optional[str]
     # Boolean specifying if token can be used
     is_enabled: Optional[bool]
+    # Token value (only sudo tokens can use this)
+    token: Optional[str]
```

Comment:
Do you have a corresponding pipeline-top PR for this? What's the use case here?

Reply:
APIv1 migration

Reply:
I need to be able to set user's tokens to what they were in APIv1